external-crates/move/crates/move-compiler/src/parser/mod.rs

```diff
@@ -152,8 +179,7 @@ fn parse_file(
     MatchedFileCommentMap,
     FileHash,
 )> {
-    let mut f = File::open(fname.as_str())
-        .map_err(|err| std::io::Error::new(err.kind(), format!("{}: {}", err, fname)))?;
+    let mut f = vfs.join(canonicalize(&fname))?.open_file()?;
```

Comment:
Why do we need both the fname and the path at this point? Can't we have just the path?